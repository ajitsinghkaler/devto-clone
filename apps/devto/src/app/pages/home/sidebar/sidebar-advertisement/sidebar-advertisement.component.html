<div class="ad-widget">
  <a href="https://shop.dev.to/"
    ><img
      src="https://res.cloudinary.com/practicaldev/image/fetch/s--H03EII_F--/c_limit%2Cf_auto%2Cfl_progressive%2Cq_auto%2Cw_350/https://thepracticaldev.s3.amazonaws.com/i/yulj7krmm6s7owlk3l7d.jpg"
      style="margin-top: 15px"
      class="w-100"
      loading="lazy" /></a
  ><a href="https://shop.dev.to/">
    <h2 class="m-0 ad-decription">New Champion-Brand DEV Wear is Here!</h2>
  </a>
</div>
