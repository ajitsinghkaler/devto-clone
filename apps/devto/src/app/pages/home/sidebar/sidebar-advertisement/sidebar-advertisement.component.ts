import { Component } from '@angular/core';

@Component({
  selector: 'app-sidebar-advertisement',
  templateUrl: './sidebar-advertisement.component.html',
  styleUrls: ['./sidebar-advertisement.component.scss'],
})
export class SidebarAdvertisementComponent {}
