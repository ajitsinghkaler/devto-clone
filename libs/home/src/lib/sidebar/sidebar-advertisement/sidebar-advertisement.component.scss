.ad-decription {
  text-align: center;
  line-height: 1.29em;
}
