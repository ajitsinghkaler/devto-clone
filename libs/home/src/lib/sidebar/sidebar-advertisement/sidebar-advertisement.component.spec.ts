import { ComponentFixture, TestBed } from '@angular/core/testing';

import { SidebarAdvertisementComponent } from './sidebar-advertisement.component';

describe('SidebarAdvertisementComponent', () => {
  let component: SidebarAdvertisementComponent;
  let fixture: ComponentFixture<SidebarAdvertisementComponent>;

  beforeEach(async () => {
    await TestBed.configureTestingModule({
      declarations: [SidebarAdvertisementComponent],
    }).compileComponents();
  });

  beforeEach(() => {
    fixture = TestBed.createComponent(SidebarAdvertisementComponent);
    component = fixture.componentInstance;
    fixture.detectChanges();
  });

  it('should create', () => {
    expect(component).toBeTruthy();
  });

  it('should render', () => {
    expect(fixture).toMatchSnapshot();
  });
});
