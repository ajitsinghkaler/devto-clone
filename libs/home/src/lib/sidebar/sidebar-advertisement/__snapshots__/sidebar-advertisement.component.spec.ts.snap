// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SidebarAdvertisementComponent should render 1`] = `
<app-sidebar-advertisement
  __ngContext__={[Function LRootView]}
>
  <div>
    <a
      href="https://shop.dev.to/"
    >
      <img
        class="w-100"
        loading="lazy"
        src="https://res.cloudinary.com/practicaldev/image/fetch/s--H03EII_F--/c_limit%2Cf_auto%2Cfl_progressive%2Cq_auto%2Cw_350/https://thepracticaldev.s3.amazonaws.com/i/yulj7krmm6s7owlk3l7d.jpg"
        style="margin-top: 15px;"
      />
    </a>
    <a
      href="https://shop.dev.to/"
    >
      <h2
        class="m-0 ad-decription"
      >
        New Champion-Brand DEV Wear is Here!
      </h2>
    </a>
  </div>
</app-sidebar-advertisement>
`;
